### ISSUE TITLE
New device icon?

### ISSUE THREAD

**Issue by RoadXY** [Baldhor/Homey-ESPhome-Enhanced#64]
**Is your feature request related to a problem? Please describe.**
The device icon could use an update.
I knew there was a way for Athom to create it but couldn't remember where.

Found it!
https://athomcommunity.slack.com/archives/C04E9KJAD/p1697278443826829?thread_ts=1697212180.697519&cid=C04E9KJAD

**Comment by Baldhor** [comment]
Yeah, I was thinking about it, but they make icons based on pictures ...
and in ESPhome world ... the picture of what :)

Just for info, I also plan to go ping ESPhome and tell them to also give reference to Homey, not just Home Assistant :)

**Comment by RoadXY** [comment]
Yeah, that would be nice!

**Comment by Baldhor** [comment]
@RoadXY Did you mean the default device picture should be an svg ?

Currently:
![image](https://github.com/Baldhor/Homey-ESPhome-Enhanced/assets/57640917/9b5b0b63-87a3-4b53-a137-abe5022fec8d)

If so, it should be:
https://esphome.io/_images/logo.svg

**Comment by RoadXY** [comment]
Yes, default device picture as ESPHome logo would be better, imho.
